blog.html

```diff
@@ -14,13 +20,122 @@
 	</head>
 	<body>
 		<div id="container">
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+			<header class="clearfix">
+				<section="logo">
+				<h1>
+					Relaxr
+				</h1>
+			</section>
+			<!-- <nav class="clearfix"> -->
+				<ul class="top-nav clearfix">
+					<li><a href="#"><span class="nav">Blog</span></a></li>
+					<li><a href="#"><span class="nav">Contact Us</span></a></li>
+					<li><a href="#"><span class="nav">Team</span></a></li>
+					<li><a href="#"><span class="nav">FAQ</span></a></li>
+					<li><a href="#"><span class="nav">About</span></a></li>
+				</ul>
+			<!-- </nav> -->
 			</header>
-			<footer>
-				&copy; Footer Goes Here
+
+
+		<main class="clearfix">
+			<div class="articles clearfix">
+			<article>
+				<div class="article-header">
+				<h2>How I implemented Relaxr in 2 weeks and changed my life.</h2>
+				</div>
+				<img src="images/blog_photo1.jpg" alt="Blog Photo 1" class="article-image">
+				<div class="article-div">
+				<p class="article-body">
+					Lorem ipsum dolor sit amet, consectetur adipsiscing elit. Fusce varius augue nex metus veverra interdum. Sed in gravida lectus. Pellenteque habitant morbi tristique senectus et netu et maelsuada fames ac turpis egestas. Marius ut vulpatate nisl. Maecenas quis auctor dui. Ut porta aliquam lorem, at volutpat orci commodo a. Suspendisse nunc orci, porttitor ac sapien a, rutrum porttitor odio. Phasellus faucibus tristique orci eget dictum. Proin tincidunt vehicula leo sed mattis. Nunc faucibus augue a pellentesque tristique. Proin suscipit in dui ut convallis.
+				</p>
+			</div>
+				<span class="read-more">
+					<a href="#">Read more > </a>
+				</span>
+			</article>
+			<article>
+				<div class="h3">
```

Comment:
The class name here could be a bit better. Class names should generally give some insight into the purpose of this area, not the children it will contain.